models/iaf_psc_alpha_multisynapse.h

```diff
@@ -56,6 +56,9 @@ This can be reached by specifying separate receptor ports, each for
 a different time constant. The port number has to match the respective
 "receptor_type" in the connectors.
 
+For implementation details see the
+`IAF_neurons_singularity <../model_details/IAF_neurons_singularity.ipynb>`_ notebook.
+
```

Comment:
The "If tau_m is very close to ..." note should be added in the docs here as well. The text should be changed to "... a synaptic time constant ..." instead of mentioning tau_syn_ex/in explicitly.